Codeinwp/otter-blocks
# Add API validation step for OpenAI in Settings

Closes https://github.com/Codeinwp/otter-internals/issues/191


### Summary

The PR adds a validation step to OpenAI API insertion by making an example request, thus making sure it also has the required capabilities for making a Chat Completion request.

### Screenshots 

----

### Test instructions


- Add an invalid key and confirm it shows an error about exactly what is wrong.
- Try using a valid key and confirm it is saved.
- Also trying emptying the field and confirm it is removed without showing any errors.

---- 

### Checklist before the final review
